src/services/search/SearchRequestMapper.res

```diff
@@ -0,0 +1,48 @@
+
+let mapSearchArtistToItemArtist = (artist: SearchResponse.artistItem) => {
+  let startImg: SearchResponse.image = {height: 999999, url: ""} 
+  let image = switch artist.images {
+  | None => startImg
+  | Some(a_) => a_->Array.reduce(startImg, (a, b) => a.height < b.height? a:b )
```

Comment:
tbh I am not sure about using the smallest image anymore, looks kinda too small in the UI. Is there a default size which is always available?